.scalafmt.conf

```diff
@@ -0,0 +1,3 @@
+version = 2.7.5
```

Comment:
I'm not familiar with this file type. Was it put here by the IDE?

Reply:
no, it is the code style plugin for Scala....it is the most commonly used one and needs a config file